plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/groovy/ParserConfiguration/config.jelly

```diff
@@ -7,10 +7,13 @@
         <f:entry title="${%Parsers}" description="${%title.description}">
           <i:repeatable field="parsers"/>
         </f:entry>
+        <f:entry field="consoleLogScanningPermitted" title="${%Permit Parsing of Console?}" description="${%consoleLogScanningPermitted.description}">
```

Comment:
Can you please use a property here as well for the `title`, I'm not sure how the question mark works in properties files.